pilot/pkg/networking/util/util.go

```diff
@@ -121,6 +126,9 @@ var ALPNInMeshWithMxc = []string{"istio-peer-exchange", "istio"}
 // ALPNHttp advertises that Proxy is going to talking either http2 or http 1.1.
 var ALPNHttp = []string{"h2", "http/1.1"}
 
+// ALPNHttp3OverQUIC advertises that Proxy is going to talk HTTP/3 over QUIC
+var ALPNHttp3OverQUIC = []string{"h3", "h3-29"} // TODO: Add various h3-* corresponding to drafts
```

Comment:
Do we need anything other than `h3`? Seems like the old ones are just legacy, since we are greenfield deployment we can just support the official RFC?

Reply:
Easier to add than to take away :slightly_smiling_face:

Reply:
Yeah. I'll remove `h3-29`